### DISCUSSION TITLE
Request to Add Multi-Window Display Function

### DISCUSSION THREAD

**Discussion by xiaobaixuejava** [rustdesk/rustdesk#7030] [General]
I would like to request the developers to add a feature that allows users to display 2, 4, or 8 windows simultaneously on the same screen for easier multitasking and information viewing. I hope this functionality can be implemented in the next version.

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1714199082640581039

**Reply by xiaobaixuejava** [reply]
I used the new version 1.2.4 and found that the effect was not quite what I expected. I will provide you with a picture as an example. What I envisioned was to display 2, 4, or 8 connected screens in a single window, making it easier for monitoring without having to switch tabs back and forth. This way, I can work on one side and observe the dynamic screen of the remote machine in a small corner of the screen.
![image015_S](https://github.com/rustdesk/rustdesk/assets/67936232/50fc2fa5-956f-4250-aa77-4a2082e56b4e)
